### DISCUSSION TITLE
How can I make item frames face away from the block they are placed on?

### DISCUSSION THREAD

**Discussion by Flopgop** [Minestom/Minestom#1599] [Help] [answered]
yk like it does in vanilla?

**Comment by KrystilizeNevaDies** [comment, marked as answer]
You need to set the Orientation in the ItemFrameMeta AND set the entity's rotation to the same position. If they are not in sync, bugs can happen. (the classic one is where the itemframe turns to the entity rotation after 60 seconds)

If you do not know how to calculate the correct orientation, I can't help you with that atm as I am on mobile.
